pom.xml

```diff
@@ -199,6 +199,146 @@
             <version>${dep.trino.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-hive</artifactId>
```

Comment:
I don't like this, can we copy the page source classes?

Reply:
We could do that, but the page source classes are intertwined with a lot of other `trino-hive` dependencies. Essentially it's the entire folder. Do we want to proceed?

Reply:
It turns out to be a lottttt of files we need to copy...